post_installation_configuration/multi-architecture-configuration.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: ASSEMBLY
+:context: multi-architecture-configuration
+[id="post-install-multi-architecture-configuration"]
+= Configuring a multi-architecture cluster
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+A multi-architecture cluster is a cluster that supports worker machines with different architectures. You can deploy a multi-architecture cluster by creating an Azure installer-provisioned cluster using the multi-architecture installer binary. For Azure installation, see xref:../installing/installing_azure/installing-azure-customizations.adoc[Installing on Azure with customizations].
+
+[WARNING]
+====
+The multi-architecture clusters Technology Preview feature has limited usability with installing, upgrading, and running payloads. 
```

Comment:
Are there specific risks we're concerned about?  Because aside from [the generic Tech-Preview risks][1], I don't see anything specific to multi-arch clusters that makes these actions riskier than usual.

[1]: https://docs.openshift.com/container-platform/4.11/nodes/clusters/nodes-cluster-enabling-features.html#nodes-cluster-enabling-features-about_nodes-cluster-enabling

Reply:
i think the "risk" here is the limited usability like the fact that imagestreams are not manifestlisted by default, so things like `oc debug` might not work right out of the box on the non control plane arch worker node